donate/payments/tasks.py

```diff
@@ -135,7 +147,10 @@ def send_newsletter_subscription_to_basket(data):
 
 
 def send_transaction_to_basket(data):
-    donation_data = {
+    if settings.DONATION_RECEIPT_METHOD == 'DONATE':
```

Comment:
Note, we originally agreed on BASKET, SQS for the two options, but DONATE, BASKET makes more sense to me! 😄